utils/testing_provider_utils.go

```diff
@@ -0,0 +1,34 @@
+package utils
```

Comment:
If we can keep this logic somewhere in the fuzzing package or a subpackage, that'd be ideal, only because it's fuzzing specific and not a general util. But not a big deal. Just makes discovery a bit ???

Reply:
Resolved in https://github.com/crytic/medusa/commit/a2cab0f0ed9ddc46c9faa0cf72f570a06feaaa0e

Reply:
I created a fuzzing/utils package and added this file in there. I don't love it at the moment but should be good enough for now. Don't think it will matter in the long run ;)